document available host preflights types (#656)
replicatedhq/kurl.sh: src/markdown-pages/create-installer/host-preflights/block-devices.md

---
path: "/docs/create-installer/host-preflights/block-devices"
date: "2022-01-13"
weight: 29
linktitle: "Block Devices"
title: "Block Devices"
---
 

> The block devices host preflight check can be used to detect and validate the block devices attached to the machine.

Reviewer: ```suggestion
You can use the block devices host preflight check to detect and validate the block devices attached to the machine.
```

## Block Devices Collector

The `blockDevices` collector will collect information about the block devices.

### Parameters

The `blockDevices` collector accepts the [shared collector properties](https://troubleshoot.sh/docs/collect/collectors/#shared-properties).

## Block Devices Analyzer

The `blockDevices` analyzer supports multiple parameters for filtering:

`includeUnmountedPartitions`: Include unmounted partitions in the analysis. Disabled by default.<br/>

> `minimumAcceptableSize`: The minimum acceptable size to filter the available block devices by during analysis. Disabled by default.

Reviewer: ```suggestion
`minimumAcceptableSize`: The minimum acceptable size to filter the available block devices during analysis. Disabled by default.
```
